nextcord/ext/commands/bot.py

```diff
@@ -122,9 +122,9 @@ def __repr__(self):
 _default = _DefaultRepr()
 
 class BotBase(GroupMixin):
-    def __init__(self, command_prefix, help_command=_default, description=None, **options):
+    def __init__(self, command_prefix=MISSING, help_command=_default, description=None, **options):
         super().__init__(**options)
-        self.command_prefix = command_prefix
+        self.command_prefix = command_prefix if command_prefix is not MISSING else tuple()
```

Comment:
seems great, but why is it a empty tuple instead of an empty string?

Reply:
Empty string would trigger on every message (it is a prefix of all messages). Instead we want an empty non-string iterable, since that will not trigger anything (there are no prefixes in it to check).

Reply:
oh ok!